auth/src/main/java/com/firebase/ui/auth/ui/DialogBase.java

```diff
@@ -5,24 +5,38 @@
 import android.support.annotation.Nullable;
 import android.support.annotation.RestrictTo;
 import android.support.v4.app.DialogFragment;
+import android.view.ContextThemeWrapper;
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 public class DialogBase extends DialogFragment {
```

Comment:
@samtstern Could we just get rid of this class? The only dialog using it is the `RecoveryEmailSentDialog` and it doesn't use any of this. I also noticed that there's dead code in the `RecoveryEmailSentDialog`:
```java
public static void show(String email, FlowParameters parameters, FragmentManager manager) {
    // We never use the flow params
    bundle.putParcelable(ExtraConstants.EXTRA_FLOW_PARAMS, parameters);
}
```